README.md

```diff
@@ -35,11 +35,11 @@ First, write the following lines in your `.rootlogon.C`
 ```
 {
  gInterpreter->AddIncludePath("$LCIO");
- gSystem->Load("${LCIO}/lib/liblcio.so");
- gSystem->Load("${LCIO}/lib/liblcioDict.so");
+ gSystem->Load("liblcio.so");
+ gSystem->Load("liblcioDict.so");
 }
 ```
-and add $LCIO/lib to your LD_LIBRARY_PATH:
+and add $LCIO/lib and/or $LCIO/lib64 to your LD_LIBRARY_PATH:
 ```
 export LD_LIBRARY_PATH=$LCIO/lib:$LD_LIBRARY_PATH
```

Comment:
```suggestion
export LD_LIBRARY_PATH=$LCIO/lib:$LCIO/lib64:$LD_LIBRARY_PATH
```
Having both paths on `LD_LIBRARY_PATH` is not the most elegant thing, but usually doesn't harm either. Non-existant paths are usually simply ignored.